### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by jcouv**
FYI @dotnet/roslyn-compiler

### CODE REVIEW

**Review by jaredpar — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*       @dotnet/roslyn-compiler
```
Comment (333fred) [inline .github/ISSUE_TEMPLATE/CODEOWNERS:1]: This should likely just be the proposals folder, not the whole repo.

Comment (jcouv) [reply .github/ISSUE_TEMPLATE/CODEOWNERS:1]: I'd be open to that. But there's been a couple of accepted PRs (mostly typos) to the specs folder as well, which is useful to get notifications for.

Comment (333fred) [reply .github/ISSUE_TEMPLATE/CODEOWNERS:1]: I don't think the whole compiler team needs to be notified for that though. Perhaps we should make a team with people who actually participate on csharplang for those types of changes? You, me, Mads, @BillWagner, and Jared?

Comment (jcouv) [reply .github/ISSUE_TEMPLATE/CODEOWNERS:1]: @dotnet/roslyn-compiler Please reply if you want to be notified as a reviewer for csharplang PRs.

Comment (RikkiGibson) [reply .github/ISSUE_TEMPLATE/CODEOWNERS:1]: I think it's no big deal to notify everyone from the compiler team. If people aren't happy about the notifications we can revisit.

**Pull request merged**